easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.8.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,43 @@
+name = 'TensorFlow'
+version = '1.8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-1.5.0_swig-env.patch',
+    'TensorFlow-%(version)s_icc-fixes.patch',
+    'TensorFlow-1.8.0_remove-msse-hardcoding.patch',
+    'TensorFlow-1.6.0_use_64b_alignment.patch',
+    'TensorFlow-1.7.0_remove_DATE_defines.patch',
+    'TensorFlow-1.8.0_no_grpc.patch',
+    'TensorFlow-1.8.0_no_boringssl.patch',
+]
+checksums = [
+    '47646952590fd213b747247e6870d89bb4a368a95ae3561513d6c76e44f92a75',  # v1.8.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.5.0_swig-env.patch
+    '15528ced7932cdb3286c60ac47c00de044e456ac5ddb828a888d107dcd5586dc',  # TensorFlow-1.8.0_icc-fixes.patch
```

Comment:
@akesandgren Seems like this should be 05f8e5c186407a03dd325bfc969d745959b5fa12a92dad3dba8168beb4b8da98?